### DISCUSSION TITLE
Old problem--small menu font

### DISCUSSION THREAD

**Discussion by dadams1** [freeplane/freeplane#740] [Help]
Hello Freeplane Community,

Firstly, thanks to the developers for making this great tool available. I use this tool on several machines. On one, running windows 10, the menu font size is minute and I can't seem to fix it. The version is 1.10.3. Tools-->preferences-->menu item font size does not change the font. Adjusting screen size similarly does not change the font size. Any suggestions? 

Thanks!
<img width="454" alt="freeplane font" src="https://user-images.githubusercontent.com/5521842/195359391-0556e328-e44c-4be2-ab6f-26e20680b68b.png">

**Comment by euu2021** [comment]
Did you restart FreePlane?

Please, try the solutions found at #505 and #345

**Comment by dpolivaev** [comment]
Try Freeplane 1.10.4

**Comment by wora55** [comment]
1.10.4 did the job for me (after restart!)

**Comment by dadams1** [comment]
Thanks for the help. Upgrading from 1.10.3 to 1.10.6 did not help. Adding the l4jini additions did help. The app looks goofy (all the icons on the left are huge), but at least the menus are readable if I set the values to 2.

**Reply by dpolivaev** [reply]
Upgrade to 1.10.6u1
